packages/core/src/App/Containers/VerificationModal/verification-modal.scss

```diff
@@ -24,6 +26,141 @@
+                                    margin-right: 1.6rem;
+                                    display: flex;
+                                    flex-wrap: wrap;
+                                    .account-poa__upload-box {
+                                        border: 1px solid var(--icon-grey-background);
+                                        border-radius: $BORDER_RADIUS * 2;
+                                        margin: 0 0.8rem 1.6rem;
+                                        padding: 0.8rem;
+                                        display: flex;
+                                    }
+                                }
+                                &-box {
+                                    flex-direction: column;
+                                    margin-bottom: 2rem;
+                                    text-align: center;
+                                    align-items: center;
+                                    justify-content: flex-start;
+                                    flex: 1 0 calc((100% / 3) - 3.2rem);
+                                }
+                                &-file {
+                                    flex: 1;
+                                    height: 24rem;
+                                    position: relative;
+                                    margin: 0 0.8rem;
+
+                                    .dc-file-dropzone {
+                                        border: 1px solid var(--general-active);
+                                    }
+                                }
+                                &-remove-btn {
+                                    position: absolute;
+                                    width: 1.6rem;
+                                    height: 1.6rem;
+                                    top: 0.8rem;
+                                    right: 0.8rem;
+                                    cursor: pointer;
+                                    transition: transform 0.25s linear;
+                                }
+                            }
+                            &__details {
+                                &-section {
+                                    margin-top: 1rem;
+                                    display: flex;
+                                    flex-wrap: wrap;
+                                }
+                                &-fields {
+                                    width: 100%;
+                                    min-width: 40rem;
+                                    flex: 1;
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            &__fieldset {
+                margin-bottom: 0;
+                .dc-autocomplete {
+                    margin-bottom: 3.2rem;
+                }
+            }
+            &__header {
+                font-size: var(--text-size-xs);
+                font-weight: bold;
```

Comment:
Do we need to set  font-size and weight. I see default values are same.
can you double check?